### PR TITLE
Upgrade babel to 7 and use preset-env

### PR DESCRIPTION
**Pull request by bdo**

### CODE REVIEW

**Review by alexjfno1 — APPROVED** [review]
Nice work 👍

File: package.json [hunk 1]
```diff
@@ -46,10 +45,10 @@
     "supertest": "^3.3.0"
   },
   "dependencies": {
+    "@babel/core": "^7.1.2",
+    "@babel/node": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
```
Comment (ldabiralai) [inline package.json]: These are dev dependencies

**Review by ldabiralai — CHANGES_REQUESTED** [review]
Good work, one comment

File: package.json (hunk 1 above)
Comment (bdo) [reply package.json]: Ah ok, I thought that we needed them for the simulado binary...

**Comment by ldabiralai** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  :tada: Your code is formatted correctly
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: .babelrc [hunk 2]
```diff
@@ -1,3 +1,12 @@
 {
-  "presets": ["es2015", "stage-0"]
-}
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "current"
```
Comment (ldabiralai) [inline .babelrc]: At the moment we support node 8, we should change this to 8 so it's not a breaking change.

**Pull request merged**